pkg/cluster/internal/create/actions/config/config.go

```diff
@@ -112,12 +112,13 @@ func (a *Action) Execute(ctx *actions.ActionContext) error {
 		return err
 	}
 	if len(workers) > 0 {
-		kubeadmClusterConfig.ControlPlane = false
```

Comment:
if we have any workers this always caused all of the nodes to get false for this, this commit fixes it